packages/dd-trace/src/config.js

```diff
@@ -515,6 +515,7 @@ class Config {
     this._setValue(defaults, 'traceId128BitGenerationEnabled', true)
     this._setValue(defaults, 'traceId128BitLoggingEnabled', false)
     this._setValue(defaults, 'tracing', true)
+    this._setValue(defaults, 'apmTracingEnabled', true)
```

Comment:
I was tempted to name it `apmTracing` because of the `tracing` property but as it is a bool i think it's better to add the `Enabled` suffix

Reply:
agreed, but let's sync with APM on that

Reply:
The suffix adds clarity. Let's keep it.